cypress/integration/seededFlows/moderationFlows/adjustPostTags.spec.js

```diff
@@ -100,6 +100,9 @@ describe('Adjust post tags', () => {
   });
 
   describe('from article page', () => {
+    // Helper function for pipe command
+    const click = ($el) => $el.click();
```

Comment:
The `pipe` command should be passed a named function to allow it to show up in logs properly